src/sentry/utils/email/address.py

```diff
@@ -0,0 +1,55 @@
+import re
+from email.utils import parseaddr
+
+from django.utils.encoding import force_bytes
+
+from sentry import options
+
+from .signer import _CaseInsensitiveSigner
+
+# Pull email from the string: u'lauryn <lauryn@sentry.io>'
+EMAIL_PARSER = re.compile(r"<(.*)>")
```

Comment:
This looks like it's from `/src/sentry/integrations/vsts/webhooks.py`, why is it in here? I don't see it being used anywhere.

Reply:
It got caught up in splitting the PR. I'll take it out.